File: 01/uvod.py
### Proměnná, přiřazení proměnné, výraz

a = 3
b = 5

print("a = ", a)
print("b = ", b)

c = a + b
d = a + b
print(d)

print( "Proměnná d:", d )
print( f"Proměnná d: {d}\n" )

d = 0.024
print( "d = ", d , "\n")

## Složitějš výrazy
import numpy as np

e = 0.01*np.sqrt( ( a + b ) + 1 ) + np.sin( np.rad2deg( np.pi/2 ) )

f = 9 / 1*( 0.5 + 2.5 )         # pozor na poradi zavorek!
print( "f = ", f, " určeno výrazem: f = 9 / 1*( 0.5 + 2.5 )" )
f = 9 / ( 1*( 0.5 + 2.5 ) )     # pozor na poradi zavorek!
print( "f = ", f, " určeno výrazem: f = 9 / ( 1*( 0.5 + 2.5 ) )" )

g = 9 / 1+( 0.5 + 2.5 )         # pozor na poradi zavorek!
print( "g = ", g, " určeno výrazem: g = 9 / 1*( 0.5 + 2.5 )" )
g = 9 / ( 1+( 0.5 + 2.5 ) )     # pozor na poradi zavorek
print( "g = ", g, " určeno výrazem: g = 9 / ( 1+( 0.5 + 2.5 ) )" )

File: 01/uvod.m
%%% Proměnná, přiřazení proměnné, výraz

a = 3       % vyraz neukonceny strednikem se automatiky vypisuje do konzole
b = 5;      % vyraz ukonceny strednikem se priradi, ale nevypise

disp("__________________________________________________")

c = a + b
d = a + b;
disp(d)     % pro manualni vypis do konzole souzi prikaz 'disp(@argument)'

disp("__________________________________________________")

disp( "Proměnná d: " ), disp( d )   % manuální výstup doplnění o komentář
fprintf( "Proměnná d: %d\n", d )    % formátkovaný výstup
disp("")                            % vypis prazdny radek (pro oddeleni vystupu)

d = 0.024   % prepsani hodnoty vyrazu

disp("__________________________________________________")

%% Složitějš výrazy
e = 0.01*sqrt( ( a + b ) + 1 ) + sin( pi/2 )
e = 0.01*sqrt( ( a + b ) + 1 ) + sin( deg2rad( 90 ) )
e = 0.01*( ( a + b ) + 1 )^0.5  % jiny vyraz pro mocninu (argument)^(mocnina)

disp("__________________________________________________")

f = 9 / 1*( 0.5 + 2.5 )         % pozor na poradi zavorek!
f = 9 / ( 1*( 0.5 + 2.5 ) )     % pozor na poradi zavorek!

disp("__________________________________________________")

g = 9 / 1+( 0.5 + 2.5 )         % pozor na poradi zavorek!
g = 9 / ( 1+( 0.5 + 2.5 ) )     % pozor na poradi zavorek
